Problem_2.py

```diff
@@ -0,0 +1,24 @@
+# 448. Find All Numbers Disappeared in an Array
+
+# code:
+
+class Solution:
+    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
+        res = [0]*len(nums)
+        
+        # iterated over all the elements in nums and filled in the res array
+        for i in range(len(nums)):
+            res[nums[i]-1] += 1
+        
+        # For the elements in res where the elements are not updated, that's the result
+        store = []
+        for i in range(len(res)):
+            if res[i]==0:
+                store.append(i+1)
+                
+        return store
+
+# Time Complexity: O(n)
+# Space Complexity: O(1)
```

Comment:
I see that you are creating another res array. Try modifying the input array to save on space.